aws/main.tf

```diff
@@ -0,0 +1,60 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 5.0"
+    }
+  }
+}
+
+locals {
+  yaml_secrets= yamldecode(file("../secrets.yaml"))
+}
+
+# Configure the AWS Provider
+provider "aws" {
+  region = "us-east-2"
+}
+
+resource "aws_ssm_parameter" "github_pat" {
+  name  = "/github/pat"
```

Comment:
Just a thought, can we create a variable for prefix and use it for all the parameters?

Reply:
Sure! maybe `/test` or `/demo`

Reply:
Sure, the default value of the variable could be `/demo`